aws/resource_aws_ssm_parameter_test.go

```diff
@@ -183,6 +202,16 @@ resource "aws_ssm_parameter" "foo" {
 `, rName, value)
 }
 
+func testAccAWSSSMParameterBasicConfigChangeName(rName string, value string) string {
```

Comment:
What is the reason for having a copy of `testAccAWSSSMParameterBasicConfig` with a different name instead of just passing a different name as argument? 🙂